### PR TITLE
Update railway extension

### PR DESCRIPTION
**Pull request by danieldeichfuss**
## Description

- Fixed broken links in the actions (the URL structure of railways must have been changed, all except for the settings URLs were broken)
- Changed default action link to Architecture (the default page of projects in Railway)

## Screencast

https://github.com/user-attachments/assets/01141c83-4f6e-4c74-bca0-a982a54552f4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/railway/src/railway.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@ const backboardUrl = "https://backboard.railway.app/graphql/v2";
 export const railwayWebUrl = "https://railway.app";
 
 export const projectUrl = (projectId: string, page?: string): string =>
-  `${railwayWebUrl}/project/${projectId}/${page ?? "settings"}`;
+  `${railwayWebUrl}/project/${projectId}/${page ?? ""}`;
 
-const token = getPreferenceValues<Preferences>().railwayApiKey;
```
Comment (xmok) [inline extensions/railway/src/railway.ts:10]: Why remove the `Preferences` type? It enforces "token" as string in TS.

File: extensions/railway/src/railway-projects.tsx [hunk 2]
```diff
@@ -45,22 +45,21 @@ const ListProjects: React.FC = () => {
...
+                  url={projectUrl(p.id, "observability")}
+                  shortcut={{ modifiers: ["cmd"], key: "o" }}
                 />
                 <Action.OpenInBrowser
-                  title="Project Metrics"
-                  url={projectUrl(p.id, "metrics")}
-                  shortcut={{ modifiers: ["cmd"], key: "m" }}
+                  title="Project Logs"
+                  url={projectUrl(p.id, "logs")}
+                  shortcut={{ modifiers: ["cmd"], key: "l" }}
```
Comment (xmok) [inline extensions/railway/src/railway-projects.tsx:62]: Should we mention the new keyboard shortcuts in `CHANGELOG`? It's a change for those who use shortcuts a lot.

File: extensions/railway/src/railway.ts (hunk 1 above)
Comment (danieldeichfuss) [reply extensions/railway/src/railway.ts:10]: Sorry, my bad. It was throwing an error and it looked like it worked the same without it, but my TS Server must have been stuck. Changed it back and it works now.

File: extensions/railway/src/railway-projects.tsx (hunk 2 above)
Comment (danieldeichfuss) [reply extensions/railway/src/railway-projects.tsx:62]: Yes, absolutely. Added it ✅. Just fyi, all the links and shortcuts I changed were 404s, so I thought changing them would do no harm.

Comment (xmok) [reply extensions/railway/src/railway-projects.tsx:62]: Yup that's fine. I don't have any active projects in Railway rn so I can't test it but the video makes it seems like all's well 👍.

File: extensions/railway/src/railway.ts (hunk 1 above)
Comment (xmok) [reply extensions/railway/src/railway.ts:10]: Happens to me all the time. I use the command palette in VS Code to `restart TS server` then it usually starts working again.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**